packages/typo3-docs-theme/resources/public/css/theme.css

```diff
@@ -18123,7 +18123,7 @@ textarea.form-control-lg {
   display: none !important;
 }
 
-.shadow {
+.shadow, img.with-shadow, figure.with-shadow {
```

Comment:
here too

Reply:
This css is automatically generated, it does not make sense to review it unless you are tinkering with the tools who generate it?